Header header

geometry_msgs/Twist pose_error
geometry_msgs/Twist twist_error

geometry_msgs/Twist last_pose_meas
geometry_msgs/Twist last_pose_desi
geometry_msgs/Twist last_twist_meas
geometry_msgs/Twist last_twist_meas_filtered
geometry_msgs/Twist last_twist_desi
geometry_msgs/Wrench last_wrench_desi

float64[] desired_torque
float64[] measured_torque
